sdk/test/UnitTests/Custom/RuntimeDependencyRegistryTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Amazon;
+using Amazon.RuntimeDependencies;
+using Amazon.SecurityToken;
+using Moq;
+using Amazon.Runtime.SharedInterfaces;
+using Amazon.Runtime.Internal;
+using Amazon.Runtime.SharedInterfaces.Internal;
+using AWSSDK.Runtime.Internal.Util;
+using Amazon.Runtime.Internal.Auth;
+using Amazon.KeyManagementService;
+using Amazon.SSO;
+using Amazon.SSOOIDC;
+
+namespace AWSSDK.UnitTests
+{
+    [TestClass]
+    public class RuntimeDependencyRegistryTests
+    {
```

Comment:
nit: not necessary to fix since they are just test cases, but there are lot of instances where `registered` is spelled `registerd` and `instance` is spelled `Instanace`